src/tactics/Board.js

```diff
@@ -2013,7 +2011,7 @@ export default class Board {
     });
   }
   isAdjacentToHighlighted({ x, y }, includeDiagonal = true) {
-    return [
+    return undefined != [
```

Comment:
this ensures we get a Boolean. Not strictly necessary, but might prevent hard-to-trace bugs in the future.

Reply:
I usually use "!!" to cast values to boolean, although the idiomatic way of casting values to boolean is to wrap them with Boolean().  I'll make the change to use "!!" after merging your work here shortly.